apps/web/src/server/views/applications/case/applications-case-layout.njk

```diff
@@ -0,0 +1,51 @@
+{% extends "app/layouts/app.layout.njk" %}
+
+{% from "./applications-case-menu.component.njk" import applicationsCaseMenu %}
+{% from "govuk/components/accordion/macro.njk" import govukAccordion %}
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "../components/status-tag.component.njk" import statusTag %}
+
+{% set pageCaption = displayValue %}
+{% set pageTitle = application.title %}
+
+{% block beforeContent %}
+	{{ govukBackLink({
+		text: "Back to applications",
+		href: '/applications-service/' + domainType
+	}) }}
+{% endblock %}
+
+{% block pageHeading %}
+	<div class="govuk-grid-row govuk-!-padding-top-9 govuk-!-padding-bottom-4" style="background-color: AliceBlue">
```

Comment:
better to remove inline style. also the blue pageHeading looks quite different from the figma.
you could create a new scss in the style/7-components folder and style it without inline tweaks